Automattic/newspack-custom-content-migrator
# fix(Attachment): sanize filename before search query

## How to test
- Upload an image with a special character in the filename (e.g. `1*test.jpeg`) using the `import_external_file` method two times and notice that the image is duplicated in the media library.
- Pull this and redo the previous  step and notice that the media is not duplicated this time.

---
